package.json

```diff
@@ -27,19 +27,22 @@
         "prepublishOnly": "npm run build",
         "serve": "npx http-serve storybook-static",
         "start": "npm run storybook",
-        "storybook": "start-storybook -p 8080"
+        "storybook": "start-storybook -p 8080",
+        "bundle-size": "npm run build && bundle-size measure"
     },
     "dependencies": {
         "keyborg": "^1.0.7",
         "tslib": "^2.1.0"
     },
     "devDependencies": {
+        "@azure/data-tables": "^13.0.0",
         "@babel/core": "^7.16.0",
         "@babel/plugin-proposal-class-properties": "^7.16.0",
         "@babel/plugin-transform-react-jsx": "^7.12.7",
         "@babel/plugin-transform-typescript": "^7.16.0",
         "@babel/preset-react": "^7.12.7",
         "@babel/preset-typescript": "^7.12.7",
+        "@fluentui/bundle-size": "^1.1.2",
```

Comment:
These extra dev deps should be removed once microsoft/fluentui#20612 is release right ?

Reply:
Yes, this is temporary

Reply:
Downgrading this version to the one before azure-tables integration would be good enough as a temp workaround to #83